packages/graphql_flutter/pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: graphql_flutter
 description: A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package.
-version: 5.0.2-beta.7
+version: 5.1.0
 homepage: https://github.com/zino-app/graphql-flutter/tree/master/packages/graphql_flutter
 
 dependencies:
-  graphql: ^5.0.2-beta.6
+  graphql: ^5.0.1
```

Comment:
Do you think I makes sense to release graphql before graphql_flutter, so we can get the right version here?

Reply:
yes it can have sense, more sense IMHO that put it in a single release

Reply:
if you agree I will split in two this PR and release the graphql before

Reply:
That sounds great!

Reply:
Ok graphq PR is ready